sqlglot/dialects/mysql.py

```diff
@@ -24,6 +25,9 @@ class MySQL(Dialect):
         "%S": "%S",
     }
 
+    class Tokenizer(Tokenizer):
+        QUOTES = ["'", '"']
```

Comment:
why did you add this? isn't " an identifier?

Reply:
It looks like for strings both are supported but for date and datetime literals you need single quotes:
```
Single quotes should be used for string values like in the VALUES() list. Double quotes are supported by MySQL for string values as well, but single quotes are more widely accepted by other RDBMS, so it is a good habit to use single quotes instead of double.

MySQL also expects DATE and DATETIME literal values to be single-quoted as strings like '2001-01-01 00:00:00'. Consult [the Date and Time Literals](https://dev.mysql.com/doc/refman/5.7/en/date-and-time-literals.html) documentation for more details, in particular alternatives to using the hyphen - as a segment delimiter in date strings.
```
https://stackoverflow.com/questions/11321491/when-to-use-single-quotes-double-quotes-and-backticks-in-mysql